### PR TITLE
Fix darkstar and overlord awacs to be flying for longer and more cons…

### PR DESCRIPTION
**Pull request by pingoocoder**
Changed:

- Remove 'Required Modules'
- Set Darkstar immortal (was only previously invisible) and option to ignore enemies to avoid diving and losing radar coverage
- Set Krymsk overlord awacs to rearm and refuel at krymsk to continuously provide radar coverage. Previously was just landing and not providing coverage after that

### CODE REVIEW

**Review by MacMillan-M — APPROVED** [review]
Changes look good. Wasnt tested in game by reviewer. Good to merge if Pingoo was able to QA.

**Comment by pingoocoder** [comment]
Will test this first locally before merging.

**Pull request merged**
